### ISSUE TITLE
Error occurs with enum

### ISSUE THREAD

**Issue by B0bbyD0llar** [olvlvl/composer-attribute-collector#25]
Hi,

I am having problems reading my route attributes. One property of the route class is an enum, apparently this is not defined correctly in attributes.php.

Error Message:

```
Error: Class "olvlvl\ComposerAttributeCollector\APP\Core\RouteMethod" not found
```

Example:

```php
35 => 
    array (
      0 => 
      array (
        'path' => '/management/pod/del/[*:id]',
        'method' => 
        APP\Core\RouteMethod::All,
        'name' => 'PoDDelete',
      ),
      1 => 'APP\\Admin\\POD\\DeleteController',
      2 => 'delete',
    ),
```
Does anyone have any idea what's going wrong?

**Comment by mnavarrocarter** [comment]
Hey @B0bbyD0llar, I'm not the maintainer of this package but I've stumbled upon the same issue.

The library does capture enums correctly (the last commit even has test cases that cover it). The issue is with PHP native `var_export` (that this library uses to generate the dump). Apparently, it does not export the enum with the required `\` at the start. This is a bug on PHP itself, that is fixed on 8.2.0. If you are using anything lower than that, unfortunately, it won't work.

You can check the reproducible behavior here:

https://onlinephp.io/c/ee23b

**Comment by mnavarrocarter** [comment]
Actually, I'll raise a PR because there is something that can be done to fix this from the mantainer's POV.
